resource/out

```diff
@@ -81,6 +83,15 @@ if [ -n "$version_file" ]; then
   version_arg=--version=$(cat "$version_file")
 fi
 
+has_image_arg=""
+if [ "${has_image}" == "true" ]; then
+  has_image_arg=--has-image
+fi
+if [ "${has_image}" == "false" ]; then
+  has_image_arg=--no-has-image
+fi
```

Comment:
If `has_image` is not passed I'd rather not passing the option to the shell command, so that the default only lives in one place (the python code `has_image = True`) and not in 2 places.
If that's what you're referring to